3600/3/.gitignore

```diff
@@ -0,0 +1 @@
+db/*.sqlite3
```

Comment:
My mistake `db/*.sqlite` to exclude database file

Reply:
It's my mistake too